doc/source/_toc.yml

```diff
@@ -163,6 +163,7 @@ parts:
               - file: serve/deploying-serve
               - file: serve/monitoring
               - file: serve/performance
+              - file: serve/autoscaling
```

Comment:
I don't think this should be in the top-level TOC, only the user guides.

Reply:
All listed here is under `- file: serve/user-guide` if you see more above .